extensions/sonarr/package.json

```
@@ -0,0 +1,92 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "sonarr",
+  "title": "Sonarr",
+  "description": "View your Sonarr instance, upcoming shows and more",
+  "icon": "command-icon.png",
+  "author": "Aayush9029",
+  "categories": [
+    "Media",
+    "Other"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Sonarr Shows",
```

To keep it consistent with our commands structure I would change this to start with the verb and put Sonarr in the subtitle.
```suggestion
      "title": "View Upcoming Shows",
```